hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -195,13 +202,19 @@ public void setStateVersion(int stateVersion) {
 	@Override
 	public MerkleNetworkContext copy() {
 		setImmutable(true);
+
+		List<DeterministicThrottle.UsageSnapshot> copyUsageSnapshots = new ArrayList<>();
+		for(DeterministicThrottle.UsageSnapshot snapshot: usageSnapshots){
+			copyUsageSnapshots.add(new DeterministicThrottle.UsageSnapshot(snapshot.used(), snapshot.lastDecisionTime()));
```

Comment:
`UsageSnapshot`s are immutable, so there's no need to create a new instance here.